contracts/lib/Approvable.sol

```diff
@@ -3,18 +3,19 @@ pragma solidity ^0.4.11;
 import './Ownable.sol';
 
 contract Approvable is Ownable {
-  mapping(address => bool) approved;
+  mapping(address => bool) public approved;
 
   function Approvable() {
-    approveAddress(msg.sender);
+    updateAddressState(msg.sender, true);
   }
 
   modifier onlyApproved() {
     require(approved[msg.sender]);
     _;
   }
 
-  function approveAddress(address _validAddress) onlyOwner {
-    approved[_validAddress] = true;
+  function updateAddressState(address _address, bool _approved) onlyApproved {
```

Comment:
Would prefer independent add and remove functions. How do you feel about `approve(address _address)` and `revokeApproval(address _address)`?